spec/controllers/registrations_controller_spec.rb

```diff
@@ -113,4 +114,65 @@
       end
     end
   end
+
+  context 'update user info' do
+    let(:user2) { create(:user) }
+    let(:user3) { build(:user) }
+    before do
+      sign_in user2
+    end
+    it 'checks if user is updated' do
+      post :update, params: {
+        user: {
+          name: user3.name,
+          email: user3.email,
+          password: user3.password,
+          password_confirmation: user3.password,
+          current_password: user2.password
+        }
+      }
+      expect(flash[:notice]).to eq I18n.t('devise.registrations.update_needs_confirmation')
+      expect(user2.reload.name).to eq(user3.name)
+      user2.reload
```

Comment:
Is this reload necessary? If it is needed for confirm then you can just call `user2.reload.confirm`, and if it is needed in the expect block below then it can just be put inline there as well.